packages/devtools_app/web/index.html

```diff
@@ -52,6 +52,22 @@
       });
     }
   </script>
+
+  <!-- Script to fail gracefully when CanvasKit crashes. -->
+  <script>
+    var realWarn = window.console.warn.bind(console);
+    window.console.warn = function(obj) {
+      if( typeof(obj) == "string" && obj.indexOf("CanvasKit") != -1 && obj.indexOf("font") != -1) { 
+        window.alert(
+          "The Flutter CanvasKit Skia Backened has encountered a fatal error and needs to restart. " +
```

Comment:
Either:

```
¯\_(ツ)_/¯
```

or:

```
The Flutter web backend ('CanvasKit') has encountered a fatal error and needs to restart. ...
```

Reply:
done. used the second. Also fixed an issue with multiple alerts because of cascading errors from canvaskit before the reload can happen.